orttraining/orttraining/python/training/ortmodule/_torch_module_ort.py

```diff
@@ -175,3 +161,11 @@ def module(self):
         # to save and load a complete checkpoint
 
         return self._original_module
+
+    def __setstate__(self, state):
+        # Attempt to deserialize torch module
```

Comment:
The `# Attempt` comments make me believe there will be an exception handling when an attempt fail :) Are these comments needed?